doc/requirements.rst

```diff
@@ -213,6 +202,16 @@ Test coverage
     :splitsourcetype:
     :stats:
 
+.. item-piechart:: Test coverage chart targeting ITEST only
```

Comment:
```suggestion
.. item-piechart:: Test coverage chart targeting ITEST only with matrix attribute containing filtered labels
```

Reply:
Improved title in 682c6222802b68dd65013926b5f7b4a04ebc0b4d